src/main/java/Duke.java

```diff
@@ -6,5 +9,78 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        String intro = "Hello! I'm Duke \n" +
+                    "What can I do for you? \n";
+        System.out.println(intro);
+        List<Task> ls = new ArrayList<>();
+        while(true) {
```

Comment:
As per the coding standard, consider adding a space between `while` and `(true)`, so that it becomes `while (true)`.